lib/backends/webgl/ops/conv.ts

```diff
@@ -288,15 +289,15 @@ export class WebGLConv extends Conv {
     return outputShape;
   }
   protected calcSharedDimReadSize(sharedDim: number): number {
-    const preferredBatchSize = 16;
+    const preferredBatchSize = 8;
     if (sharedDim < preferredBatchSize || sharedDim % preferredBatchSize !== 0) {
       return sharedDim;
     }
     return preferredBatchSize;
   }
   protected calcBlockSize(outputLayout: TextureLayout): [number, number]|undefined {
-    const preferredRowCount = 64;
-    const preferredColCount = 64;
+    const preferredRowCount = 32;
```

Comment:
how do we determine the best block size? does it depend on the GPU spec

Reply:
Purely experimental.
These are the best combinations I could come up with on my machine.
In the future, we may decide to expose these to the very savvy users